docs/generated/manifests/nx-api.json

```diff
@@ -434,6 +434,701 @@
         "type": "generator"
       }
     },
+    "migrations": {
+      "/nx-api/angular/migrations/remove-library-generator-simple-module-name-option": {
```

Comment:
I think these should be sorted from latest to earliest. We might also want to make the `package.json` updates more clear, like `20.2.0 (update packages)` rather than just `20.2.0`.